src/client/components/LeaderboardList/LeaderboardList.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { withKnobs, text, number, object } from '@storybook/addon-knobs';
+import ima from '../../assets/images/avatar1-ce.png';
+import LeaderboardItem from './LeaderboardItem';
+
+export default {
+  title: 'Test Leaderborad Component',
+  decorators: [withKnobs],
+};
+
+export const TestLeaderboradComp = () => {
```

Comment:
Small spelling mistake: TestLeaderBoardComp. 
I would also go for the full naming TestLeaderBoardComponent, according to the Naming conventions on "Prioritize readability over writability", but this is just a suggestion. :)

Reply:
yes good idea

Reply:
Nice job Amer, but in this case I think we should name the file as this example from project decomentary : MotorcycleCard.component.js to be sure at this comes from components folder not from container folder, and the function also as what they write.
I hope experte correct what we commit....